src/main/java/com/hotels/bdp/circustrain/bigquery/context/CircusTrainBigQueryConfiguration.java

```diff
@@ -103,6 +111,20 @@ private GoogleCredentials getCredential(GCPSecurity gcpSecurity) throws IOExcept
     }
   }
 
+  @Bean
+  @Order(0)
+  CopierFactory supportedSchemeCompositeCopierFactory(
+      List<CopierFactory> copierFactories,
+      BigQueryDataExtractionManager bigQueryDataExtractionManager) {
+    BigQueryCopierFactory bigQueryCopierFactory = new BigQueryCopierFactory(bigQueryDataExtractionManager);
+    List<CopierFactory> delegates = new ArrayList<>();
+    delegates.add(bigQueryCopierFactory);
```

Comment:
So the fact that the delegates are a List and this is being put in first means it gets chosen first?

Reply:
Yes